designsafe/static/scripts/projects/components/manage-project/manage-project.template.html

```diff
@@ -519,14 +519,16 @@ <h3 class="text-center">
                         </button>
                     </div>
                 </div>
-                <!-- FR Fields -->
-                <div ng-if="$ctrl.form.projectType === 'field_recon'">
+                <!-- NH Fields -->
```

Comment:
Once you've made your adjustments, you'll need to checkout master, pull the latest commits, and then rebase this branch over it. Your changes here will probably have a conflict with Sarah's merged changes.